cloudevents.md

```diff
@@ -87,160 +386,906 @@ The *configuration change* event is sent when a desired state for a service is a
+    "stage": "staging",
+    "teststrategy": "performance",
+    "deploymentstrategy":"blue_green_service",
     "evaluationdetails": {
-      "options": {
-        "timeStart": 1558428643,
-        "timeEnd": 1558429200
-      },
-      "totalScore": 50,
-      "objectives": {
-        "pass": 90,
-        "warning": 75
-      },
       "indicatorResults": [
         {
-          "id": "ResponseTime_Service",
-          "violations": [],
-          "score": 50
-        },
-        {
-          "id": "FailureRate_Service",
+          "score": 0.5,
+          "value": {
+              "metric": "request_latency_p95",
+              "value": 1.162000,
+              "success": true
+          },
           "violations": [
             {
-              "value": 12.55862428348098,
-              "key": "SERVICE-34E5F883CB9DF269",
-              "breach": "upperSevere",
-              "threshold": 10
+              "criteria": "<=+10%",
+              "targetValue": 0
             }
           ],
-          "score": 0
+          "status": "warning"
+        },
+        {
+          "score": 2,
+          "value": {
+              "metric": "error_rate",
+              "value": 0,
+              "success": true
+          },
+          "violations": null,
+          "status": "pass"
         }
       ],
-      "result": "fail"
-      }
-   }
+      "result": "warning",
+      "score": 83.34,
+      "timeEnd": "2019-11-05T16:35:27.152Z",
+      "timeStart": "2019-11-05T16:30:27.152Z"
+    }
+  }
+}
+```
+</p>
+</details>
+
+([&uarr; up to index](#keptn-cloud-events))
+
+## Problem Open
+
+The *problem open* event is sent when a monitored service causes any problem.
+
+### type
+```json
+"type": "sh.keptn.event.problem.open"
+```
+
+### data
+```json
+"ProblemEventData": {
+  "required": [
+    "ProblemID",
+    "ProblemTitle",
+    "ProblemDetails",
+    "PID"
+  ],
+  "properties": {
+    "ImpactedEntities": {
+      "type": "string",
+    },
+    "PID": {
+      "type": "string",
+    },
+    "ProblemDetails": {
+      "items": {
+        "type": "integer"
+      },
+      "type": "array"
+    },
+    "ProblemID": {
+      "type": "string"
+    },
+    "ProblemTitle": {
+      "type": "string",
+    },
+    "State": {
+      "type": "string"
+    },
+    "Tags": {
+      "type": "string"
+    },
+    "project": {
+      "type": "string"
+    },
+    "service": {
+      "type": "string"
+    },
+    "stage": {
+      "type": "string"
+    }
+  },
+  "additionalProperties": false,
+  "type": "object"
+}
+
+```
+
+### Example
+<details><summary>Example of sh.keptn.event.problem.open</summary>
+<p>
+
+```json
+{
+  "type": "sh.keptn.event.problem.open",
+  "specversion": "0.2",
+  "source": "https://github.com/keptn/keptn/prometheus-service",
+  "id": "f2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext": "08735340-6f9e-4b32-97ff-3b6c292bc509",
+  "data": {
```

Comment:
we now also have the fields `project`, `stage` and `service` in the `data` object